protocols/grpc/hyperstart.proto

```diff
@@ -110,20 +113,46 @@ message CreateSandboxRequest {
 message DestroySandboxRequest {
 }
 
+enum IpFamily {
```

Comment:
Is that possible to rename this to "IPFamily" without violation to ".proto" format?
I think "IP" is better than "Ip". The same for "IpAddress".
What do you think?

Reply:
@WeiZhang555 Sure, I'll fix that.